builtin/providers/google/config.go

```diff
@@ -13,6 +13,7 @@ import (
 	"golang.org/x/oauth2"
 	"golang.org/x/oauth2/google"
 	"golang.org/x/oauth2/jwt"
+	bigquery "google.golang.org/api/bigquery/v2"
```

Comment:
That's correct, not sure why that ended up being named. I'll push a fix for that straight away.